react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { time: 0, running: false, icon: 'fa-solid fa-play' };
+    this.startTick = this.startTick.bind(this);
+  }
```

Comment:
Try reworking your solution to not store the icon in the state.

Reply:
gotcha